DZSwiftUI/DZSwiftUI/ViewModel/ClimateViewModel.swift

```diff
@@ -0,0 +1,32 @@
+// ClimateViewModel.swift
+// Copyright © RoadMap. All rights reserved.
+
+import SwiftUI
+
+/// Вью модель экрана настроек системы кондиционирования
+final class ClimateViewModel: ObservableObject {
+    // MARK: - Public Properties
+
+    @Published var isConditionerTapped = false
+    @Published var isAlertShown = false
+    @Published var temperatureProgressValue = 0.0
+    @Published var minimumTemperature = 15.0
```

Comment:
```suggestion
    @Published var minimumTemperature = 15.0
```
располагай переменные по типу,   так проще читать, числовые с числовыми,  булевые с булевыми